gazelle/manifest/manifest.go

```diff
@@ -94,11 +95,31 @@ func (f *File) Decode(manifestPath string) error {
 	return nil
 }
 
+// ModulesMapping is the type used to map from importable Python modules to
+// the wheel names that provide these modules
+type ModulesMapping map[string]string
+
+// MarshalYAML makes sure that we sort the module names before marshalling
+// the contents of `ModulesMapping` to a yaml file. This ensures that the
+// file is deterministically generated from the map
+func (m ModulesMapping) MarshalYAML() (interface{}, error) {
+	var mapslice yaml.MapSlice
+	var keys []string
+	for key := range m {
+		keys = append(keys, key)
+	}
+	sort.Strings(keys)
```

Comment:
I'd use https://github.com/emirpasic/gods/tree/dbdbadc158ae6b453820b3cfb8c6cb48be4d7ddf#treeset as it may give some performance improvement while adding the elements to the set. We already have this library as a dependency, so it's straightforward to use it.

In your approach, you make a whole copy of the slice and then sort it, which is not very optimal.

Reply:
ah sweet, didn't know the dep was already pulled in! (was trying to avoid new deps) :)
thanks for the review!